CHANGELOG.md

```diff
@@ -1,9 +1,223 @@
 # Changelog
 
+## v0.62.0
+
+This major release includes Flipper support by default, improved dark mode support, moving Apple TV to react-native-tvos, and more. See the [blog post](https://reactnative.dev/blog/2020/03/24/version-0.62) for all of the highlights.
+
+This release comes in the midst of a global pandemic. We’re releasing this version today to respect the work of hundreds of contributors who made this release possible and to prevent the release from falling too far behind master. Please be mindful of the reduced capacity of contributors to help with issues and prepare to delay upgrading if necessary.
+
+If you're upgrading, manual intervention may be required for your app. Please see the [upgrade-helper](https://react-native-community.github.io/upgrade-helper/) for a detailed breakdown of the changes required and see [this issue](https://github.com/react-native-community/releases/issues/179) for known issues.
+
+Note two known issues with their corresponding workarounds:	
+- [Release version instacrash on Android](https://github.com/facebook/react-native/issues/28163)	
```

Comment:
Btw, 👴 here, is ‘instacrash’ a word that helps people understand this sentence?

Reply:
I just copied the title of the issue, if there's a better way of phrasing it we could change it here and in the issue.

Maybe something like `Crash on Android when creating a release version`?

Reply:
What about?

```suggestion
- [Crash on launch of Android release builds](https://github.com/facebook/react-native/issues/28163)	
```

Reply:
Hmm, in retrospect, I thought this was about another issue–namely https://github.com/facebook/react-native/issues/28270.

I actually don’t think we should link to https://github.com/facebook/react-native/issues/28163, as it does not appear to be a problem affecting multiple people. All commenters seem to have different crash logs and the original poster seems to be using Vue (?).

Reply:
Alright, removed this 👍